addons/common/functions/fnc_checkFiles.sqf

```diff
@@ -63,7 +63,7 @@ private _oldCompats = [];
 } forEach _addons;
 
 if (_oldAddons isNotEqualTo []) then {
-    _oldAddons = _oldAddons apply {"%1.pbo", _x};
+    _oldAddons = _oldAddons apply { format["%1.pbo", _x] };
```

Comment:
?

Reply:
?

Reply:
This is weird. Seems like this was the intention, but have you tested it?

Reply:
I have not tested, can't atm. But I can't see what else could've possibly been the intention. @PabstMirror Can you confirm?

Reply:
I'm not Pabst, but if you look at the old code https://github.com/acemod/ACE3/commit/9d1695ffe2e1f3e0665c2f0891dfa4631f862b2b#diff-40dceb2af61593c5d04c1135da9450c7853e0fb6f8513adf7dccd9baa1102497, it only listed one outdated PBO.
I imagine it was not intentional to omit the `format`.

Reply:
I don't think I actually wrote that but fix looks good